example/ios/Demo.xcodeproj/xcshareddata/IDETemplateMacros.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>FILEHEADER</key>
+    <string>
```

Comment:
Xcode's limitation: it always uses line comment (`//`) for the first line

Reply:
great this works, regardless of comment style.